web/client/src/App.js

```diff
@@ -75,9 +76,18 @@ function App() {
           </Grid>
           <Grid item xs>
             <MaterialUI paperClass="paper-bottom">
-              <ModuleInfo title="" content={[]} hasNote={false} note="" />
+              <ModuleInfo
+                title="Using a Token in a HTTP Request"
```

Comment:
Change capitalization to match label format. I also suggest: Use your token for an HTTP request